tasks/agent_migrate.sh

```diff
@@ -22,3 +22,4 @@ if [ $? -ne 0 ]
     printf "The puppet.conf 'server' parameter was updated to %s. \n" "$TARGET_HOST"
 
 fi
+
```

Comment:
This task can be replaced by the built-in `puppet_conf` task.

Reply:
What about the ssl clean, is it not more the combination  of functions that makes this a "migrate" task

Reply:
but we do need the equivalent powershell version